tests/api.c

```diff
@@ -13083,11 +13083,12 @@ static int test_wc_AesCbcEncryptDecrypt (void)
         0x30, 0x31, 0x32, 0x33, 0x34, 0x35, 0x36, 0x37,
         0x38, 0x39, 0x61, 0x62, 0x63, 0x64, 0x65, 0x66
     };
-    byte    vector[] = /* Now is the time for all w/o trailing 0 */
+    byte    vector[] = /* Now is the time for all good men w/o trailing 0 */
     {
         0x4e,0x6f,0x77,0x20,0x69,0x73,0x20,0x74,
         0x68,0x65,0x20,0x74,0x69,0x6d,0x65,0x20,
-        0x66,0x6f,0x72,0x20,0x61,0x6c,0x6c,0x20
+        0x66,0x6f,0x72,0x20,0x61,0x6c,0x6c,0x20,
```

Comment:
Do we really need to change the test vector?

Reply:
Yes, unless I revert the `XMEMCMP()` verification of successful encrypt-decrypt to check only the first `AES_BLOCK_SIZE` bytes.  The original test was doing an unaligned encrypt, silently ignoring the final 8 bytes of the test vector, and checking only the first block of the decryption result, giving the false impression that all was well.